group-aws/integration.md

```diff
@@ -12,167 +12,28 @@ shall be considered a feature request.
 Following the 2.5 release, all new modules MUST have integration
 tests.
 
-Modules marked `preview` SHOULD have integration tests for new
-features.
+Following the 2.10 release, all modules MUST have integration tests.
```

Comment:
Given the number of modules without integration tests must still feels a little harsh.

Reply:
I'd really like to get there, but yeah there's still gaps.  Wanted to try to idea on though.

Reply:
What do you think of changing the below SHOULD (regarding bug fixes for modules that have existing tests) to a MUST?